praw/models/reddit/live.py

```diff
@@ -345,6 +345,24 @@ def discussions(self, **generator_kwargs):
         url = API_PATH['live_discussions'].format(id=self.id)
         return ListingGenerator(self._reddit, url, **generator_kwargs)
 
+    def report(self, type):  # pylint: disable=redefined-builtin
```

Comment:
Should we make 'spam' the default? Or do you think it's better that it must be explicitly provided?